src/components/Components/StoreButton.jsx

```diff
@@ -4,22 +4,8 @@ import { useClient } from 'cozy-client'
 import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 import { ButtonLink } from 'cozy-ui/transpiled/react/Button'
 import AppIcon from 'cozy-ui/transpiled/react/AppIcon'
-import { generateUniversalLink } from 'cozy-ui/transpiled/react/AppLinker'
```

Comment:
(This comment is not about the above line, it's about the file). Why is there "components" twice in the path of the file ? src/components/StoreButton.jsx is not sufficient ?

Reply:
`/components` contains all React components, and `/components/Components` only contains small reusable generic components such as buttons, spinner... do we have a common name to use instead of `Components` ?

Reply:
>  Do we have a common name to use instead of Components ?

`/components/common` ? In Banks, page like components are grouped by functionality in "ducks" folders, and there is also a `/components` folder for common/generic components.

I personally do not care much about folder location since I navigate mostly by jumping from file to file with a search palette, but here I think that `components/Components` is not the best we can do.

Reply:
I will create an other PR to not pollute this one.